<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckTable</class>
 <widget class="QWidget" name="CheckTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="checkTableBg" native="true">
     <property name="styleSheet">
      <string notr="true">#checkTableBg{
    background-color : #FFFFFF;
    /*border : 1px solid #EBEDF0;*/
}

*{
     font-family : 微软雅黑;
     font-size : 14px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="inputBtnArea" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#inputBtnArea{
    background-color : #F7F8FA;
    border-radius : 10px;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>41</number>
         </property>
         <property name="leftMargin">
          <number>23</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>27</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="videoUserIdBox" native="true">
           <property name="minimumSize">
            <size>
             <width>340</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>340</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="idLabel">
              <property name="text">
               <string>视频用户ID: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="userIdEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#userIdEdit{
    background-color : #FFFFFF;
    border-radius : 4px;
    border : 1px solid #DCDEE0;
    padding-left : 19px;
}</string>
              </property>
              <property name="placeholderText">
               <string>请输入用户ID</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="videoStatusBox" native="true">
           <property name="minimumSize">
            <size>
             <width>260</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>260</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>7</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="status">
              <property name="minimumSize">
               <size>
                <width>35</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>状态: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="videoStatus">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#videoStatus{
    background-color : #FFFFFF;
    border : 1px solid #DCDEE0;
    border-radius : 4px;
    color : #222222;
    padding-left : 16px;
    line-height : 40px;
}

#videoStatus::drop-down{
    border : none;
    width : 33px;
}

#videoStatus::down-arrow{
    width : 15px;
    background-color : transparent;
    image : url(&quot;:/images/admin/triangle.png&quot;);
}

#videoStatus QAbstractItemView{
    outline : 0px;
}

#videoStatus QAbstractItemView::item{
     color : #222222;
     padding-left : 16px;
     background-color : #FFFFFF;
     height : 30px;
}

#videoStatus QAbstractItemView::item:selected{
    background-color : #409CE1;
    color : #FFFFFF;
}
</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>300</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="btnOperatorBox" native="true">
           <property name="minimumSize">
            <size>
             <width>194</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>194</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="resetBtn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#resetBtn{
    background-color : #FFFFFF;
    color : #222222;
    border-radius : 4px;
    border : 1px solid #DCDEE0;
}</string>
              </property>
              <property name="text">
               <string>重置</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="queryBtn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#queryBtn{
    background-color : #3ECEFF;
    border-radius : 4px;
    color : #FFFFFF;
}</string>
              </property>
              <property name="text">
               <string>查询</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="tableTittleArea" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>67</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>67</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="empty" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="labelBox" native="true">
           <property name="styleSheet">
            <string notr="true">#labelBox{
     background-color : #EBFAFF;
     border-top-left-radius : 10px;
     border-top-right-radius : 10px;
     border-bottom-left-radius : 0px;
     border-bottom-right-radius : 0px;
}

#labelBox QLabel{
    font-weight : bold;
    color : #222222;
    line-height : 20px;
}</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="userIdLabel">
              <property name="minimumSize">
               <size>
                <width>156</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>156</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>视频用户ID</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="nickNameLabel">
              <property name="minimumSize">
               <size>
                <width>138</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>138</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>用户昵称</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="videoTittleLabel">
              <property name="minimumSize">
               <size>
                <width>265</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>265</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>视频标题</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="videoLabel">
              <property name="minimumSize">
               <size>
                <width>259</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>259</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>视频</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="statusLabel">
              <property name="minimumSize">
               <size>
                <width>192</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>192</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>状态</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="checkerLabel">
              <property name="minimumSize">
               <size>
                <width>133</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>133</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>审核人</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="operatorLabel">
              <property name="minimumSize">
               <size>
                <width>133</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>133</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>操作</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="videoArea" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>480</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>480</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="styleSheet">
            <string notr="true">#scrollArea
{
     border : none;
}

/*QScrollBar:vertical 是一个子控件选择器，用于选择垂直方向的整个滚动条，包括：滑块(handle)、上下箭头按钮， 滚动轨道*/
QScrollBar:vertical{
    border : none;
    background-color : #FFFFFF;
    width : 10px;
}

QScrollBar::handle:vertical
{
    border : none;
    border-radius : 5px;
    background-color : #E8E8E8;
    min-height : 134px;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical
{
     border : none;
     width : 0px;
     height : 0px;
}


</string>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>1265</width>
              <height>465</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="layout">
               <property name="leftMargin">
                <number>1</number>
               </property>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="paginatorArea" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
